src/main/cpp/t06_longest.cpp

```diff
@@ -22,5 +22,19 @@
 using namespace std;
 
 int t06_longest() {
+    string s = {};
+    int len = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'len' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594548)